.pre-commit-config.yaml

```diff
@@ -28,26 +28,22 @@ repos:
       - id: check-yaml
       - id: check-ast
   - repo: https://github.com/astral-sh/ruff-pre-commit
-    # Ruff version (Used for linting)
-    rev: v0.7.4
+    # Ruff version.
```

Comment:
Do we need to comments at all? I think it is pretty self explanatory

Reply:
copy/pasted from its docs, removed